Aptoide/aptoide-client-v8
# [APP-135] Promote social media

**What does this PR do?**

This PR aims at implementing the social media promotion feature on 3 different views: MyAccount, Editorial, AppCoinsInfoView

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] SocialMediaView.kt

**How should this be manually tested?**

Open the mentioned views and check the card. Also click on the buttons to see that it opened the correct page.
Also check for the analytics event that is being sent.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-135](https://aptoide.atlassian.net/browse/APP-135)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
